modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,228 @@
+= Topics as Iceberg Tables
+:description: Learn how to integrate Redpanda topics with Apache Iceberg.
+:page-context-links: [{"name": "Linux", "to": "manage:topic-iceberg-integration.adoc" } ]
+:page-categories: Management, High Availability, Data Replication, Integration
+:page-beta: true
+
+
+The Apache Iceberg integration for Redpanda allows you to store topic data in the cloud in Iceberg's table format. This makes your streaming data immediately available for analytical systems, such as data warehouses like RedShift, Snowflake, and Clickhouse, and big data processing platforms, such as Apache Spark and Flink, without setting up and maintaining additional ETL pipelines. 
+
+The Iceberg integration uses xref:manage:tiered-storage.adoc[Tiered Storage]. When a cluster or topic has Tiered Storage enabled, Redpanda stores the Iceberg files in the configured Tiered Storage bucket or container.  
+
+== Prerequisites
+
+* Install `rpk`.
+
+*  {empty}
+include::shared:partial$enterprise-license.adoc[]
++
+To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+
+* Enable xref:manage:tiered-storage.adoc#set-up-tiered-storage[Tiered Storage] for the topics for which you want to generate Iceberg tables.
+
+== Limitations
+
+* You can only enable Iceberg topic integration for new Redpanda topics. 
```

Comment:
@andrwng this isn't really true though.  Just because we don't backfill doesn't mean you can't enable on existing topics